internal/initialize/security_test.go

```diff
@@ -97,7 +97,7 @@ func TestRestrictedSecurityContext(t *testing.T) {
 				"Containers must be required to run as non-root users.")
 		}
 
-		assert.Assert(t, sc.SeccompProfile == nil,
+		assert.Assert(t, sc.SeccompProfile != nil,
 			"The RuntimeDefault seccomp profile must be required, or allow specific additional profiles.")
```

Comment:
🔧 We should check the value, too. I'm pretty sure it's possible to compare to an untyped literal here.

The message of other assertions appears to quote the standard. I think the following makes it clear which rule we're targeting and that we're compliant:

```suggestion
		if assert.Check(t, sc.SeccompProfile != nil) {
			assert.Assert(t, *sc.SeccompProfile.Type == "RuntimeDefault",
				"Seccomp profile must be explicitly set to one of the allowed values.")
		}
```